### PR TITLE
MEN-4601: add index to list/count devices by status

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: none

Signed-off-by: Fabio Tranchitella <fabio.tranchitella@northern.tech>

### CODE REVIEW

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline

File: store/mongo/migration_1_10_0.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+	"github.com/mendersoftware/deviceauth/model"
+)
+
+type migration_1_10_0 struct {
+	ds  *DataStoreMongo
+	ctx context.Context
+}
+
+// Up removes device and authset indexes which include raw id data
```
Comment (merlin-northern) [inline store/mongo/migration_1_10_0.go:34]: is it true that it removes?

Comment (tranchitella) [reply store/mongo/migration_1_10_0.go:34]: argh, no, wrong comment

**Review by merlin-northern — APPROVED** [review]
lgtm

**Comment by merlin-northern** [comment]
> lgtm

it will increase the data handlers count. but we know it.

**Comment by tranchitella** [comment]
Force-pushed to fix the comment in `migration_1_10_0.go`, I won't run again the QA pipeline

**Pull request merged**